Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/common/Calculations.kt

```diff
@@ -14,37 +15,37 @@ fun calculateDaysSinceMostRecentDateAtRiskLevelAtTestRegistration(
     testRegisteredAt ?: return -1
     lastDateAtRiskLevel ?: return -1
     if (lastDateAtRiskLevel.isAfter(testRegisteredAt)) return -1
-    return Days.daysBetween(
-        lastDateAtRiskLevel,
-        testRegisteredAt
-    ).days
+    return Duration.between(
```

Comment:
This should be date changes, so it should be a calculation based on local dates, not on 24h intervals

Reply:
I think the implementation is the same as before, this should also be covered by `PpaDataExtensionsTest`
Can you show me where the new implementation could get different results from the previous one?

Reply:
Some days are only 23 hours when changing to summer time. toDays() will divide by 24h to get the number of days. Two dates both with time start of the day, say 27.3.2022 and 31.3.2022 have four date changes, but only 3 with your solution (given the 27.3. has only 23 h)

Reply:
ok got it, I change the implementation using `ChronoUnit`